pallets/pooled-staking/src/lib.rs

```diff
@@ -516,5 +528,43 @@ pub mod pallet {
 
             Calls::<T>::update_candidate_position(&candidates)
         }
+
+        #[pallet::weight(0)]
```

Comment:
```suggestion
        #[pallet::weight(0)]
```
Put the benchmarked value here